.github/workflows/tests.yml

```diff
@@ -197,16 +197,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
-      ###
-      # Install bundler and yarn dependencies
-      - name: Install dependencies
+      - name: Install yarn dependencies
+        run: yarn install
+
+      - name: Setup Exercism services
```

Comment:
I assume that we want to re-run when either `bundle exec setup_exercism_config` or `bundle exec setup_exercism_local_aws` fails. I've excluded `yarn` because that could fail due to us messing up the `package.json` file, which would lead to endless re-running.